dspace-server-webapp/src/main/java/org/dspace/app/rest/exception/DSpaceApiExceptionControllerAdvice.java

```diff
@@ -190,6 +191,13 @@ protected void MissingParameterException(HttpServletRequest request, HttpServlet
                           HttpStatus.BAD_REQUEST.value());
     }
 
+    @ExceptionHandler(PasswordNotValidException.class)
+    protected void handlePasswordNotValidException(HttpServletRequest request, HttpServletResponse resp, Exception ex)
+        throws IOException {
+        String message = "password.error.robust-password";
```

Comment:
This looks incorrect to me, as it appears this is a hardcoded reference to an i18n key that's defined in `dspace-angular`. 

In my previous review, I had suggested this i18n key be defined *on the backend* (in Messages.properties).  

Otherwise, if any other client encounters this error message, they won't know what "password.error.robust-password" means.  It won't make sense to anything except our User Interface.

So, I think this message should be a human readable message...but it needs to _not be hardcoded_ in to Java code, and instead must be easily modifiable on the backend.  That all means it likely should have it's text defined in Messages.properties.